src/app/series/directives/series-basic.component.spec.ts

```diff
@@ -10,7 +10,7 @@ describe('SeriesBasicComponent', () => {
 
   cit('shows the basic series edit fields', (fix, el, comp) => {
     expect(el).not.toQuery('publish-fancy-field');
```

Comment:
might vote for wording more like "does not render until the series is loaded" or happy medium "shows the basic series edit fields once series is loaded" to clarify?